### PR TITLE
Upload chinese translation for 'contens' part

### PR DESCRIPTION
**Pull request by SuSung-boy**
All files from 'contents' folder was translated!

### CODE REVIEW

**Comment by clefourrier** [comment]
Hi! Thank you very much!
Someone internal is proofreading things and I'll merge once it's been approved :)

File: translations/zh/contents/automated-benchmarks/designing-your-automatic-evaluation.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# 设计你的自动评估任务 
+
+## 选择数据集
+做评估时，你可以选择现有的数据集 (参考 [一些评估数据集](https://github.com/huggingface/evaluation-guidebook/blob/main/contents/automated-benchmarks/some-evaluation-datasets.md) 页面) 作为测试集，也可以设计自己的数据集。有一点非常重要，请注意：**评估的结果与评估的数据集质量高度相关**。
+
+### 使用现有的数据集
+这部分强烈建议仔细阅读！
+#### 数据集需要注意的问题
+- **数据集的标注员分布如何？**
```
Comment (adeenayakup) [inline translations/zh/contents/automated-benchmarks/designing-your-automatic-evaluation.md:9]: 样本是由谁创建的？

File: translations/zh/contents/automated-benchmarks/designing-your-automatic-evaluation.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+#### 数据集需要注意的问题
+- **数据集的标注员分布如何？** 
+在我看来，按照样本的标注员素质高低，数据集质量大致排名如下：专家构建数据集 > 付费标注数据集 > 众包数据集 > MTurk 数据集。
+你可以在数据集的说明文档 (data card) 找到标注员的统计信息，可以帮助理解数据集语言多样性。
+
+- **样本是否经过其他标注员或作者的审核？** 
+你需要先弄明白：
+	- 不同标注员标注结果是否一致？
+	-  完整数据集是否经过作者审核？
+如果付给标注员薪水过低 (通常母语不是目标语言，如 AWS Mechanical Turk)，那么标注的数据集需要尤其注意：拼写错误、语法错误、或者事实错误。
```
Comment (adeenayakup) [inline translations/zh/contents/automated-benchmarks/designing-your-automatic-evaluation.md:17]: 标注员通常不是目标语言的母语使用者（例如 AWS Mechanical Turk），否则可能会出现拼写错误、语法错误或无意义的答案。

**Review by adeenayakup — COMMENTED** [review]
Hi @clefourrier - I gave a few suggestions for improvement, but overall, the article translation looks good.

**Review by clefourrier — APPROVED** [review]
Thanks @adeenayakup for the review, and thanks @SuSung-boy for the PR! :)

**Pull request merged**